sys/include/net/coap.h

```diff
@@ -40,6 +40,7 @@ extern "C" {
 #define COAP_OPT_LOCATION_PATH  (8)
 #define COAP_OPT_URI_PATH       (11)
 #define COAP_OPT_CONTENT_FORMAT (12)
+#define COAP_OPT_MAX_AGE        (14)
```

Comment:
Appearently you need to run `dist/tools/doccheck/create_pattern.sh` on this file :-/